modules/core/src/parallel.cpp

```diff
@@ -970,7 +970,7 @@ int getNumberOfCPUs_()
 
 #endif
 
-#if !defined(_WIN32) && !defined(__APPLE__)
+#if !defined(_WIN32) && !defined(__APPLE__) && !defined(CV_BAREMETAL)
 
     static unsigned cpu_count_sysconf = (unsigned)sysconf( _SC_NPROCESSORS_ONLN );
```

Comment:
@alalek  - this call to sysconf doesn't seem to fit in any of the patches I have factored out (disable filesystem and disable threads). I am going to leave it here, as the `CV_BAREMETAL` flag seems and appropriate name for it (no OS on baremetal). Alternatively, do you have a better name? `CV_DISABLE_SYSCONF` seems to be too specific.

Reply:
This is implementation of `getNumberOfCPUs()` call.
I believe whole its code can be disabled through `#if` (returns "1")

Reply:
OK - so disable the whole function body and just return 1.